sdk/keyvault/keyvault-admin/src/tracing.ts

```diff
@@ -16,6 +25,6 @@ export const createSpan: <T extends {
   span: Span;
   updatedOptions: T;
 } = createSpanFunction({
-  packagePrefix: "AdminClient",
-  namespace: "Microsoft.KeyVault"
+  packagePrefix: packagePrefix,
```

Comment:
Do you have any cases left where you want to use `createSpan` rather than just using `trace`?

Reply:
I did have a few iterator cases that were using it; however, looking at how app-config does it I think the trace calls are not right. So I fixed it to match what you have and I can get rid of this method now (and clean up some of the imports). Thanks!